server/src/termdb.matrix.js

```diff
@@ -108,6 +108,11 @@ async function getSampleData(q) {
 		// for each non dictionary term type
 		// query sample data with its own method and append results to "samples"
 		if (tw.term.type == 'geneVariant') {
+			// Add ensembl gene id to refs.byTermId
+			const aliasLst = q.genome.genedb.getAliasByName.all(tw.term.name)
+			const ensemblGeneID = aliasLst.find(a => a.alias.toUpperCase().startsWith('ENSG')).alias
+			refs.byTermId[tw.term.name] = { ensemblGeneID }
```

Comment:
unreasonable assumption. ENSG is not available for all genomes thus shouldn't do it here. if ensemblGeneID is only required for gdc logic, its maaping should be carried out in gdc specific code e.g. mds3.gdc.js